presto-main/src/test/java/com/facebook/presto/cost/TestFilterStatsCalculator.java

```diff
@@ -435,6 +450,28 @@ public void testInPredicateFilter()
                             .nullsFraction(0.0);
                 });
 
+        // Casted literals as value
+        assertExpression(
+                new InPredicate(new SymbolReference("mediumVarchar"), new InListExpression(
+                        ImmutableList.of(
+                                new Cast(new StringLiteral("abc"), MEDIUM_VARCHAR_TYPE.toString())))))
+                .outputRowsCount(4)
+                .symbolStats("mediumVarchar", symbolStats -> {
+                    symbolStats.distinctValuesCount(1)
+                            .nullsFraction(0.0);
+                });
+
+        assertExpression(
+                new InPredicate(new SymbolReference("mediumVarchar"), new InListExpression(
```

Comment:
why not to use something like `com.facebook.presto.sql.planner.iterative.rule.test.PlanBuilder#expression` it is much readable in form of SQL